docs/user-guide/SettingsFile.md

```diff
@@ -37,6 +37,72 @@ Path to your key file in a txt file.
 
 If provided and valid, RESTler will attempt to use it during the SSL handshake.
 
+### authentication: dict (default empty)
+Settings for specifying authentication. See Authentication.md for details
+
+__Token__: Can optionally provide one of {```location```, ```token_refresh_cmd```, ```module```}
```

Comment:
>__Token__: [](http://example.com/codeflow?start=0&length=10)

This should be differentiated from "location" since it's the parent property.  

#### Token : dict

For simplicity, let's just make the type 'dict' (there are some dict(dict) etc. examples elsewhere but I believe this is not necessary given all the examples in the rest of this section).

Reply:
Updating token and certificate to following format per offline conversation

#### _token_